src/Handler/LogDnaHandler.php

```diff
@@ -68,6 +69,26 @@ protected function getDefaultFormatter(): FormatterInterface
 
     public function write(array $record)
     {
+        $body = $record['formatted'];
+
+        if (mb_strlen($body) > static::LOGDNA_BYTE_LIMIT) {
+            $decodedBody = json_decode($body, true);
+
+            $body = json_encode([
```

Comment:
Is this actually going to provide enough information? I think it's worth trying to come up with a better solution here - how about
1. Removing the last item
2. Re-encode to json
3. Still exceeding the limit, go back to 1
4. Add a very short line about how many rows were removed

Reply:
I thought about going that route but do we really want a potentially long loop with lots of mb_strlens going on in the log handler? Considering this sends as much of the exception as possible i think it should give enough info

Reply:
Ok - it was difficult to tell without a test

Reply:
@maxakropolis Good point i shall add a test